pulpcore/cli/core/group.py

```diff
@@ -185,6 +198,7 @@ def role(ctx: click.Context, pulp_ctx: PulpCLIContext, group_ctx: PulpGroupConte
                 callback=null_callback,
                 help=_('Filter roles by the associated object. Use "" to list global assignments.'),
             ),
+            domain_option,
```

Comment:
Should we add a guard that certain commands can only be used in the default domain?

Reply:
Possibly, currently Pulp should disallow this, but having a nice error message from the CLI would be helpful.